client/asset/eth/polygon_bridge.go

```diff
@@ -0,0 +1,452 @@
+package eth
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+	"strings"
+	"time"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+
+	"decred.org/dcrdex/dex"
+	"decred.org/dcrdex/dex/dexnet"
+	"decred.org/dcrdex/dex/networks/erc20"
+	"decred.org/dcrdex/dex/networks/erc20/polygonbridge"
+)
+
+var (
+	usdtEthID, _     = dex.BipSymbolID("usdt.eth")
+	usdtPolygonID, _ = dex.BipSymbolID("usdt.polygon")
+	wethPolygonID, _ = dex.BipSymbolID("weth.polygon")
+	ethID, _         = dex.BipSymbolID("eth")
+	polygonID, _     = dex.BipSymbolID("polygon")
+	ethMaticID, _    = dex.BipSymbolID("matic.eth")
+)
+
+var polygonBridgeSupportedAssets = map[dex.Network]map[uint32]uint32{
+	dex.Mainnet: {
+		usdtEthID:     usdtPolygonID,
+		usdtPolygonID: usdtEthID,
+		ethID:         wethPolygonID,
+		wethPolygonID: ethID,
+		ethMaticID:    polygonID,
+	},
+	dex.Testnet: {
+		ethID:         wethPolygonID,
+		wethPolygonID: ethID,
+		ethMaticID:    polygonID,
+	},
```

Comment:
I guess we only have one of these up right now, so can't test on testnet?

Reply:
We only have matic.eth enabled/Registered on testnet currently correct?

Reply:
We have POL on Polygon, and ETH/WETH as well.

Reply:
Oh ok, I was reading this wrong forever, its a map of maps, and those can be bridged. We can do from matic.eth to polygonID